wallet/wallet.go

```diff
@@ -2524,6 +2524,57 @@ func (w *Wallet) GetTicketsPrecise(f func([]*TicketSummary, *wire.BlockHeader) (
 	return nil
 }
 
+// GetTicketInfo returns the ticket summary and the corresponding block header
+// for the provided ticket. The ticket summary comprises of the transaction
+// summmary for the ticket and the spender (if already spent) and the ticket's
+// current status.
+func (w *Wallet) GetTicketInfo(chainClient *dcrrpcclient.Client, hash *chainhash.Hash) (*TicketSummary, *wire.BlockHeader, error) {
+	const op errors.Op = "wallet.GetTicketInfo"
+
+	ticketSummary := new(TicketSummary)
+	blockHeader := new(wire.BlockHeader)
```

Comment:
These two should be `var ...` instead of newl (so you don't unnecessarily allocate memory for these structs which will ultimately be unused given that you'll replace them with whatever makeTicketSummary returns)